tests/tt_eager/python_api_testing/unit_testing/misc/test_moreh_bmm.py

```diff
@@ -33,8 +33,9 @@ def test_moreh_bmm(shape, device):
 
     # tt bmm
     cpu_layout = ttl.tensor.Layout.ROW_MAJOR
+    tt_output_tensor = None
     tt_out = (
-        ttl.operations.primary.moreh_bmm(tt_input, tt_mat2)
+        ttl.operations.primary.moreh_bmm(tt_input, tt_mat2, tt_output_tensor)
         .cpu()
```

Comment:
Please add a test showing that the optional output tensor does not even need to be provided.  And one that shows that the optional output tensor was provided and returned back by the op.